tests/test_spectral_embed.py

```diff
@@ -314,6 +346,21 @@ def test_unconnected_warning(self):
             lse = LaplacianSpectralEmbed()
             lse.fit(A)
 
+    def test_embedding(self):
+        epsilon = 0.1
+        nodes_per_community = 100
+        P = np.array([[0.8, 0.2], [0.2, 0.8]])
+        undirected, labels_ = sbm(2 * [nodes_per_community], P, return_labels=True)
+
+        oos_idx = 0
+        A, a = remove_vertices(undirected, indices=oos_idx, return_removed=True)
+
+        lse = LaplacianSpectralEmbed(n_components=2)
+        X_hat = lse.fit_transform(A)
+        w = lse.transform(a) / nodes_per_community
```

Comment:
out of curiosity, why are we dividing by `nodes_per_community`?

Reply:
That was a minor mistake and has been corrected.